canonical/juju-dashboard
# Use application layout components

## Done

- Copy over application layout components.
- Replace custom elements with application layout components.

## QA

- Click through as much of the UI as possible and make sure nothing looks broken (some things will be visually different but that's OK).

## Screenshots

<img width="1508" alt="Screenshot 2023-11-14 at 2 13 45 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/5622feed-3da9-4c04-b5bd-95b22488abfc">
